tests/wasi-wast/wasi/unstable/unix_open_special_files.wast

```diff
@@ -0,0 +1,5 @@
+(wasi_test "unix_open_special_files.wasm"
+  (map_dirs "/dev:/dev")
+  (assert_return (i64.const 0))
+  (assert_stdout "13\n")
+)
```

Comment:
newline please

Reply:
these are generated files, I can regenerate them all but it'd probably be best to do that separately